collections/_content/resilience.md

```diff
@@ -9,18 +9,18 @@ color: "teal"
 menu-group: "1"
 intro: "<p>This year brought sobering changes in how public media is supported. Loss of funding and directional shifts in the industry are blows to efforts to make content that serves the public interest and the civic health of our communities. This turbulent landscape impacts PRX, but our driving force remains the same. PRX was created two decades ago to pursue strategic innovation by building digital solutions for public media. Our resolve continues to find new paths forward as we face the challenges of the moment.</p>"
 quotes:
-  - text: "<p>Every time there's a door closed, I feel PRX is the place that opens another one. And I, I, I think that's unique and I, again, I don't think any other organization is going to take the chances, nor have the credibility in the market, to innovate on the scale that we do. We have to get to a place at some point, where mission-driven work is supported in this country. Otherwise it's gonna be a very boring environment, and arts and culture will suffer as a result. We have to be strong and continue to be agile in trying to bring new stuff to the fore and try to be the best partners in doing so.</p>"
+  - text: "Every time there's a door closed, I feel PRX is the place that opens another one. And I think that's unique. And I, again, I don't think any other organization is going to take the chances, nor have the credibility in the market, to innovate on the scale that we do. We have to get to a place at some point, where mission-driven work is supported in this country. Otherwise it's going to be a very boring environment, and arts and culture will suffer as a result. So I think we have to be strong and continue to be agile in trying to bring new stuff to the fore and try to be the best partners in doing so."
```

Comment:
@indraprx FYI

changing "gonna" to "going to" breaks the transcript word playback. I'll upload a new transcript.

Also the transcript doesn't include "so I think"

either way I'll update the transcript.